nativescript-angular/directives/list-view-comp.ts

```diff
@@ -217,7 +217,7 @@ export class ListViewComponent implements DoCheck, OnDestroy, AfterContentInit {
 }
 
 function getSingleViewRecursive(nodes: Array<any>, nestLevel: number): View {
```

Comment:
Can we make this function public and export it so that it can be consumed from 3rd parties